### PR TITLE
Adds Preonic keymap for muzfuz. Small Planck addition.

### PR DESCRIPTION
**Pull request by muzfuz**
Adds new keymap for muzfuz's Preonic, plus a small addition to the current Planck layout.

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/preonic/keymaps/muzfuz/config.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/preonic/keymaps/muzfuz/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```

File: keyboards/preonic/keymaps/muzfuz/config.h [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#define CONFIG_USER_H
```
Comment (drashna) [inline keyboards/preonic/keymaps/muzfuz/config.h]:
```suggestion
```

File: keyboards/preonic/keymaps/muzfuz/config.h [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/preonic/keymaps/muzfuz/config.h]: This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant. 

```suggestion
```

File: keyboards/preonic/keymaps/muzfuz/config.h [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+   - Octave shift and transpose
+   - Virtual sustain, portamento, and modulation wheel
+   - etc.
+*/
+//#define MIDI_ADVANCED
+
+/* override number of MIDI tone keycodes (each octave adds 12 keycodes and allocates 12 bytes) */
+//#define MIDI_TONE_KEYCODE_OCTAVES 2
+
+#endif
```
Comment (drashna) [inline keyboards/preonic/keymaps/muzfuz/config.h]:
```suggestion
```

**Comment by muzfuz** [comment]
@drashna thank you for the review, I have made the requested changes.

It's worth noting that I copied this file from the `default` setup and didn't change it, so these things are still present in there.

**Comment by drashna** [comment]
Thanks!

And yeah, a lot of the default keymaps don't have these changes, and haven't been updated to include them, yet.

**Pull request merged**